Admin Menu: Show update count in expendable sidebar menus

obenland opened the pull request:
#### Changes proposed in this Pull Request

* Adds support for `count` prop to expandable sidebar menus

#### Testing instructions
* Install an outdated plugin on an Atomic site: https://downloads.wordpress.org/plugin/wp-search-suggest.4.zip
* WP's update check should display a count in the Plugins menu item. If it doesn't, visit `/wp-admin/update-core.php`.
* Load Calypso for that atomic site and check if that update count shows up there, too.

#### After
Nav-unification Disabled | Nav-unification Enabled
-------|------
![](https://cln.sh/IL75Vs+) | ![](https://cln.sh/nRN6VB+)

obenland committed:
Admin Menu: Show update count in expendable sidebar menus

cpapazoglou commented on client/layout/sidebar/style.scss:
```diff
@@ -233,13 +233,30 @@
 		flex: 1 1 0;
 	}
 
+	.count {
+		margin-left: 8px;
+		color: var( --color-sidebar-menu-hover-text );
+		background-color: var( --color-sidebar-menu-hover-background );
+		border-color: var( --color-sidebar-menu-hover-background );
+		line-height: 0;
+		padding: 8px 6px;
+		position: absolute;
+		top: 9px;
+	}
+
```
Can we consolidate all this count rules with these https://github.com/Automattic/wp-calypso/blob/9e259071948215c8269ff383ef9b2087d55118f6/client/layout/sidebar/style.scss#L144-L158
and probably nest them to `.sidebar {}`?

Also, it seems that can we entirely remove `position` and `top`.

The above will help us avoid this situation where we have quite the same rules clashing and an other counter misplaced.
![](https://cln.sh/vSzTNA+)

obenland replied:
Yes, please feel free to make those changes. I was unfamiliar with how that file i supposed to be structured and didn't want to step on anyone's toes.

cpapazoglou replied:
Done, I also updated the description with nav-unification disabled status.

Commit:
Consolidates .count changes:
- since .cound is now nested to the sitebar item we don't need positioning.
- let line-height , padding fallback to the count component defatuls.
